### PR TITLE
[App Insights/Operational Insights] Add GET /query operation, fix operationId inconsistency

### PR DESCRIPTION
**Pull request by alexeldeib**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/operationalinsights/data-plane/Microsoft.OperationalInsights/stable/v1/OperationalInsights.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
         }
       },
       "post": {
-        "operationId": "Query",
+        "operationId": "Query_Execute",
```
Comment (sergey-shandar) [inline specification/operationalinsights/data-plane/Microsoft.OperationalInsights/stable/v1/OperationalInsights.json]: The changes in stable API are potential SDK breaking changes

**Comment by sergey-shandar** [comment]
@alexeldeib please, resolve this issue https://travis-ci.org/Azure/azure-rest-api-specs/jobs/422715930

File: specification/operationalinsights/data-plane/Microsoft.OperationalInsights/stable/v1/OperationalInsights.json (hunk 1 above)
Comment (alexeldeib) [reply specification/operationalinsights/data-plane/Microsoft.OperationalInsights/stable/v1/OperationalInsights.json]: I had been doing work to clean up the operation groups as part of a general effort towards stable/ v1.0.0 versions of the data plane SDKs (we have preview versions for C#, Node, Java, and Python). This isn't strictly necessary and I could revert it, but is in line with what I've done in the other api-versions/with App Insights as well.

**Comment by alexeldeib** [comment]
@sergey-shandar I fixed a few issues, OAV is failing to validate this line in the spec however, which I haven't changed in months: https://github.com/Azure/azure-rest-api-specs/blob/e850b4eb50ef6a7ac2d19ddae14c53cd2dad7a5e/specification/applicationinsights/data-plane/microsoft.insights/preview/2018-04-20/swagger.json#L1081

I think this is an issue with OAV though? The property is clearly defined, even if the validation can't see it. Could it be an issue parsing the @ in the key?

**Comment by sergey-shandar** [comment]
@alexeldeib yes, it's OAV issue. I will check if I can fix it.

**Comment by sergey-shandar** [comment]
@alexeldeib I will provide a fix for OAV to unblock the PR.

**Comment by sergey-shandar** [comment]
@alexeldeib I've fixed the semantic validation tool, it works now https://travis-ci.org/Azure/azure-rest-api-specs/builds/423166589?utm_source=github_status&utm_medium=notification

There are still model validation issues https://travis-ci.org/Azure/azure-rest-api-specs/jobs/423166602

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**